hazelcast/src/main/java/com/hazelcast/client/config/ClientFailoverConfig.java

```diff
@@ -31,18 +31,32 @@
 import static com.hazelcast.client.config.ClientConnectionStrategyConfig.ReconnectMode.OFF;
```

Comment:
Not related to these changes, but while you are on it, could you also fix the line here https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/client/impl/clientside/FailoverClientConfigSupport.java#L228

It should be something like `network:outboundPorts` instead of `network:smartRouting`

Reply:
Sure. Addressed in ce976c31e4e9bd3a6920d42540a62fcdd12b1f83